rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/JsonPropertyEditor.java

```diff
@@ -0,0 +1,27 @@
+package pro.taskana.common.rest;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.beans.PropertyEditorSupport;
+import java.net.URLDecoder;
+
+public class JsonPropertyEditor extends PropertyEditorSupport {
+
+  private final ObjectMapper objectMapper;
+  private final Class<?> requiredType;
+
+  public JsonPropertyEditor(ObjectMapper objectMapper, Class<?> requiredType) {
+    this.objectMapper = objectMapper;
+    this.requiredType = requiredType;
+  }
+
+  @Override
+  public void setAsText(String text) throws IllegalArgumentException {
+    if (text != null && !text.isEmpty()) {
+      try {
+        setValue(objectMapper.readValue(URLDecoder.decode(text, "UTF-8"), requiredType));
+      } catch (Exception e) {
+        throw new IllegalArgumentException(e);
```

Comment:
Unfortunately this exception is not caught by our `TaskanaRestExceptionHandler`. I've looked at this for about an hour and then gave up. 
As far as I can tell this is because the exception is thrown during the parameter binding process of spring and thus our controller method is never called. Therefore the `TaskanaRestExceptionHandler` does not trigger.

This means that our REST API will return an empty response with status code `400`.. This is horrible, but I am clueless.